pkgs/applications/networking/mailreaders/mutt/default.nix

```diff
@@ -14,6 +14,7 @@
 , imapSupport  ? true
 , withSidebar  ? true
 , gssSupport   ? true
+, writeScript
 }:
 
 assert headerCache  -> gdbm       != null;
```

Comment:
We should drop the asserts.

Reply:
Dropped the asserts and removed top level `with lib;`.